indigo_app/views/documents.py

```diff
@@ -49,9 +49,15 @@ def get_context_data(self, **kwargs):
 
         context['document_content_json'] = json.dumps(doc.document_xml)
 
-        context['amendments_json'] = json.dumps(
-            WorkAmendmentSerializer(context={'request': self.request}, many=True)
-            .to_representation(doc.work.amendments))
+        # add 'numbered_title_localised' to each amendment
```

Comment:
My thinking was to add this to `indigo_api.serializers.WorkSerializer`, then it's included everywhere. You can just add `numbered_title` to the `fields` list on that serializer.

Reply:
I've done that as well, this is to localise the numbered title for the annotations only.

Reply:
Ah, I see.

Reply:
I wonder if we're trying to be too smart, but let's see how it goes.